linux_os/guide/services/ssh/file_groupowner_sshd_config/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+title: 'Verify Group Who Owns SSH Server config file'
+
+description: |-
+    {{{ describe_file_group_owner(file="/etc/ssh/sshd_config", group="root") }}}
+
+rationale: |-
+    The /etc/ssh/sshd_config file needs to be protected from unauthorized changes by
+    non-privileged users.
```

Comment:
Please change the `rationale` to the following:

```
rationale: |-
    Service configuration files enable or disable features of their respective services that if configured incorrectly
    can lead to insecure and vulnerable configurations. Therefore, service configuration files should be owned by the
    correct group to prevent unauthorized changes.
```